src/js/profiling/utils.ts

```diff
@@ -48,7 +48,7 @@ export function findProfiledTransactionsFromEnvelope(envelope: Envelope): Event[
 
       // @ts-expect-error accessing private property
       // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      if (event && event.contexts && event.contexts['profile'] && event.contexts['profile']['profile_id']) {
+      if (event.contexts?.['trace']?.['data']?.['profile_id']) {
```

Comment:
Are the eslint flags still required for this if condition?

Reply:
Yes, still are, when I remove them I can't make TS happy and ESlint happy, but if I overlooked something, please let me know. I would like to remove the comments!